Source/Classes/PINRemoteImageManager.m

```diff
@@ -281,6 +282,16 @@ - (void)setValue:(nullable NSString *)value forHTTPHeaderField:(nullable NSStrin
     });
 }
 
+- (void)setRequestConfiguration:(PINRemoteImageManagerRequestConfigurationHandler)configurationBlock {
```

Comment:
@garrettmoon sorry for the confusion, but are you saying we *do* need to a getter, or we don't since we can just use the direct ivar access in `downloadDataWithURL:...`. I made that direct ivar change and it's the only place this variable is accessed currently.